inc/sterling/sterling.php

```diff
@@ -0,0 +1,391 @@
+        'PT Sans Narrow, sans-serif'                        => 'PT+Sans+Narrow',
+        'Rajdhani, sans-serif'                              => 'Rajdhani:300,400,500,600',
+        'Raleway, sans-serif'                               => 'Raleway:200,300,400,500,700',
+        'Roboto, sans-serif'                                => 'Roboto:100,300,400,500',
+        'Roboto Condensed, sans-serif'                      => 'Roboto+Condensed:400,300,700',
+        'Shadows Into Light, cursive'                       => 'Shadows+Into+Light',
+        'Shrikhand, cursive'                                => 'Shrikhand',
+        'Source Sans Pro, sans-serif'                       => 'Source+Sans+Pro:200,400,600',
+        'Teko, sans-serif'                                  => 'Teko:300,400,600',
+        'Titillium Web, sans-serif'                         => 'Titillium+Web:400,200,300,600,700,200italic,300italic,400italic,600italic,700italic',
+        'Trirong, serif'                                    => 'Trirong:400,700',
+        'Ubuntu, sans-serif'                                => 'Ubuntu',
+        'Vollkorn, serif'                                   => 'Vollkorn:400,400i,700',
+        'Voltaire, sans-serif'                              => 'Voltaire',
+        
+    );
+    
+    return $font_family_array;
+}
+/**
+ * Returns all posts as an array.
+ * Pass true to include Pages
+ * 
+ * @param boolean $include_pages
+ * @return array of posts
+ */
+function sterling_all_posts_array( $include_pages = false ) {
+    
+    $posts = get_posts( array(
+        'post_type'        => $include_pages ? array( 'post', 'page' ) : 'post',
+        'posts_per_page'   => -1,
+        'post_status'      => 'publish',
+        'orderby'          => 'title',
+        'order'            => 'ASC',
+    ));
+    $posts_array = array(
+        'none'  => __( 'None', 'sterling' ),
+    );
+    
+    foreach ( $posts as $post ) :
+        
+        if ( ! empty( $post->ID ) ) :
+            $posts_array[ $post->ID ] = $post->post_title;
+        endif;
+        
+    endforeach;
+    
+    return $posts_array;
+    
+}
+
+/**
+ * Creates header using images from Custom Header
+ * 
+ */
+function sterling_get_header_panel() { ?>
+    
+    <div id="header-panel" class="container-fluid" style="background: url(<?php header_image(); ?>) no-repeat center">
+        
+        <div class="row">
+            
+            <div id="header-panel-content">
+                
+                <h1>Wedding Photography</h1>
+                
+                <div id="header-panel-links">
+                    
+                    <a>Home</a>
+                    <a>All Posts</a>
+                    <h4>Wedding Photography</h4>
+                    
+                </div>
+                
+            </div>
+            
+        </div>
+        
+    </div>
+    
+<?php }
+
+/**
+ * Creates blog post output
+ * 
+ */
+function sterling_get_blog_posts() {
```

Comment:
index.php is a WordPress template, meaning it has access to all blog posts by default within "the loop"

Calling `WP_Query` within a base template is not the correct way to display posts. This is simply achieved by 
`while ( have_posts() ) : the_post();`